src/components/Work/Work.tsx

```diff
@@ -19,6 +19,8 @@ const WorkDetail: React.FC<{ workResult: WorkResult }> = (props) => {
   //Edition Card Preprocessing
   const authorsList = EditionCardUtils.getAuthorsList(work.authors);
 
+  // FIXME: This will show 'EDD' until Mike updates the backend to change the sorting (RO => EDD => None)
```

Comment:
This isn't a front-end issue, so there's no need to mention it here.  To the extent that this isn't the desired functionality when hooked up to a real back-end, that's documented in tickets and in confluence.